src/Uno.Toolkit.UI/Behaviors/StyleExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+#if IS_WINUI
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml;
+#else
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+#endif
+
+namespace Uno.Toolkit.UI
+{
+	public static class StyleExtensions
```

Comment:
👋 I changed my mind again 🙃, let's name this `ResourceExtensions` since this isn't really specific to Style Setters